string robot_name
string grasped_object_in
---
int8 result
string grasped_object_out
string result_string

